### DISCUSSION TITLE
Perceval Connection, or lack therof

### DISCUSSION THREAD

**Discussion by mjgaughan** [sailuh/kaiaulu#237] [Q&A] [answered]
Hello! I am having some issues linking my installation of Perceval (in a Python venv) to Kaiaulu. Though I believe that I have specified the correct Perceval path in `tools.yml`, I am still running into test failures and have so far been unable to run any functions like `parse_gitlog`. Is there anything I'm overlooking when setting up the library or its dependencies? I appreciate any help or insight on the matter!

**Comment by carlosparadis** [comment, marked as answer]
Hi, 

Could you paste here how your tools.yml look like? Also, could you try to execute the Kaiaulu Architecture Notebook, i.e. `http://itm0.shidler.hawaii.edu/kaiaulu/articles/kaiaulu_architecture.html` and let me know if it runs? That will confirm if the package was compiled and loaded properly and isolate the issue to the connection to Perceval. 

Thanks!
